### ISSUE TITLE
The value of "primary ipv4" cannot be removed from a given VM in web UI

### ISSUE THREAD

**Issue by mabazyar** [digitalocean/netbox#1886]
### Issue type
[ ] Feature request 
[X ] Bug report     
[ ] Documentation   

### Environment
* Python version:  3.5.2
* NetBox version:  2.2.9

### Description

The primary-IPv4 value already assigned to a given VM can not be removed, but through the REST API.

**Comment by jeremystretch** [comment]
Thank you for taking the time to report a bug. We'll need more information to investigate this issue. Per the [contributing guidelines](https://github.com/digitalocean/netbox/blob/develop/CONTRIBUTING.md) please ensure that you have included all of the following:

* The environment in which NetBox is running
* The exact steps that can be taken to reproduce the issue
* Any error messages generated
* Screenshots (if applicable)

Please update your issue to include all of the information listed above. If no response is received within a week, this issue will be closed. Thanks!

**Comment by mabazyar** [comment]
### Issue type
[ ] Feature request
[X ] Bug report
[ ] Documentation
### Environment
Python version: 3.5.2
NetBox version: 2.2.9
OS: Centos 7

I have got a Netbox-2.2.9 instance installed on a Centos 7 supported by Python-3.5.2. 

### Description

1. Create a new VM instance.
2. Create an interface and get it attached to the VM.
3. Assign a predefined IP address to the interface.
4. Mark the IP address as "primary_ipv4" in the VM's entries.

In this case, there is no way to withdraw the "primary_ipv4" through web UI, and as a workaround, I did it by leveraging the relative API.
Like so:
```
curl -X PATCH -k --header 'Content-Type: application/json' -H "Authorization: Token 1234" -d '{"primary_ip4": null}' https://netbox.test.lan/api/virtualization/virtual-machines/8539/
```

**Comment by mabazyar** [comment]
Confirming the issue is fixed in v2.2.10.
Thanks @jeremystretch
